tools/scripts/ffi.toml

```diff
@@ -165,8 +165,6 @@ script = '''
 cd ffi/diplomat/js/examples/node
 exec --fail-on-error npm ci
 exec --fail-on-error npm test
-cd ../fixed_decimal
```

Comment:
We should probably make this work too

Reply:
ok, I think it's fine for `wasm-demo` itself to serve as the test that the npm package is working